.github/workflows/test.yml

```diff
@@ -9,8 +9,7 @@ jobs:
     strategy:
       matrix:
         version:
-          - '1.0'
-          - '1.4'
+          - '1.6'
```

Comment:
If this package declares compatibility to 1.0, we should test it.

A convenient usage is `1` that points to the latest stable 1.x release, so it will be `1.6` at the moment, but will be, say, `1.7` a few months later.

Reply:
Yeah, you're right. The reason I removed it is because the doctests fail on older versions because printing changes between versions, and I was being lazy. So maybe I should add back your tests to the regular tests, and make doctests only run on a recent version of Julia.

Reply:
Ok, tested on 1.0 now